src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredService.test.tsx

```diff
@@ -259,6 +259,6 @@ describe('Monitored Service list', () => {
 
     expect(refetchServiceCountData).toBeCalledTimes(2)
     expect(screen.queryByText(`cv.monitoredServices.showingServiceAtRisk`)).not.toBeInTheDocument()
-    expect(screen.queryByText('cv.monitoredServices.youHaveNoMonitoredServices')).toBeInTheDocument()
+    expect(screen.queryByText('cv.monitoredServices.youHaveNoMonitoredServices')).not.toBeInTheDocument
```

Comment:
Are we missing `()` at the end here?
```suggestion
    expect(screen.queryByText('cv.monitoredServices.youHaveNoMonitoredServices')).not.toBeInTheDocument()
```